src/cls/rgw/cls_rgw_types.h

```diff
@@ -638,22 +638,22 @@ struct rgw_bucket_category_stats {
 WRITE_CLASS_ENCODER(rgw_bucket_category_stats)
 
 enum cls_rgw_reshard_status {
-  CLS_RGW_RESHARD_NONE        = 0,
-  CLS_RGW_RESHARD_IN_PROGRESS = 1,
-  CLS_RGW_RESHARD_DONE        = 2,
+  CLS_RGW_RESHARD_NOT_RESHARDING  = 0,
+  CLS_RGW_RESHARD_IN_PROGRESS     = 1,
+  CLS_RGW_RESHARD_DONE            = 2,
 };
 
 static inline std::string to_string(const enum cls_rgw_reshard_status status)
 {
   switch (status) {
-  case CLS_RGW_RESHARD_NONE:
-    return "CLS_RGW_RESHARD_NONE";
+  case CLS_RGW_RESHARD_NOT_RESHARDING:
+    return "Not resharding";
     break;
   case CLS_RGW_RESHARD_IN_PROGRESS:
-    return "CLS_RGW_RESHARD_IN_PROGRESS";
+    return "Resharding in-progress";
```

Comment:
It's ultimately up to you, Mark, but I would prefer "Resharding in progress" (i.e., no hyphen), as I think it's more correct. 

In case anyone cares why, I think these phrases are versions of other phrases with the word "is" elided, as in "is not resharding", "resharding is in progress", and "resharding is done". And we would not use the hyphen in that case. We would use a hyphen when we wanted to treat "in progress" as a one word adjective, as in "status: in-progress".